src/main/java/com/softserveinc/ita/mmakoviichuk/utils/runners/ElementsUtil.java

```diff
@@ -1,14 +1,15 @@
 package com.softserveinc.ita.mmakoviichuk.utils.runners;
 
-import com.codeborne.selenide.Condition;
 import com.codeborne.selenide.SelenideElement;
 import lombok.experimental.UtilityClass;
 
+import static com.codeborne.selenide.Condition.attribute;
+
 @UtilityClass
 public class ElementsUtil {
 
-    public static void waitForAttributeChanges(SelenideElement selenideElement, String attribute, String value) {
-        selenideElement.shouldBe(Condition.attribute(attribute, value));
+    public static void expectedAttribute(SelenideElement selenideElement, String attribute, String value) {
```

Comment:
if it using only in one place, better use it there and without a separate method for such conditions

$x(XPath).shouldHave(attribute(attribute, value));